dask_cloudprovider/aws/ecs.py

```diff
@@ -254,7 +260,9 @@ async def start(self):
                         "overrides": {
                             "containerOverrides": [
                                 {
-                                    "name": "dask-{}".format(self.task_type),
+                                    "name": self.task_def_name
+                                    if self.task_def_name
+                                    else "dask-{}".format(self.task_type),
```

Comment:
Is this branch ever going to be run? It looks like the task name defaults to `dask-worker-{uuid}` further down?

Reply:
in the current workflow probably not. I made a change so that the `task_def_name` is a bit more generalized. Can be reused for both worker and scheduler container definitions